source/includes/steps-atlas-terraform-file-from-template.yaml

```diff
@@ -0,0 +1,242 @@
+---
+title: "In |vscode-short|, create a new file."
+ref: create-file-vscode
+level: 4
+content: |
+
+  .. list-table::
+     :header-rows: 1
+     :widths: 33 33 34
+
+     * - Operating System
+       - Method
+       - Actions
+
+     * - Any
+       - |vscode-short| Menu
+       - :guilabel:`File` > :guilabel:`New File`
+
+     * - MacOS
+       - Keyboard Shortcut
+       - Press ``Command`` + ``N``
+
+     * - Windows and Linux
+       - Keyboard Shortcut
+       - Press ``Control`` + ``N``
+
+---
+title: "Save the file to a new directory. Name the file ``main.tf``"
+ref: change-language
+level: 4
+content: |
+
+  .. note::
+
+     ``main.tf`` is the recommended filename for the entry point to a
+     Terraform module.
+
+---
+title: "In the ``main.tf`` file, type ``atlas``, then press the **Enter** or **Return** key."
+ref: bootstrap-atlas
+level: 4
+content: |
+
+  This populates the ``main.tf`` file with an example configuration 
+  using the `MongoDB Atlas
+  <https://www.terraform.io/docs/providers/mongodbatlas/index.html>`__ 
+  Terraform provider to create a Shared Tier |service| cluster. 
+
+  Update the example configuration to configure your cluster.
```

Comment:
Is there a good reference we can link to that gives more info on how to configure a cluster?

Reply:
Looks like this may just be the next step. If so can we delete this?

Reply:
updated